static/icons.css

```diff
@@ -0,0 +1,102 @@
+.icon {
+  display: inline-block;
+  background: transparent 50% 50% no-repeat;
+}
+
+.icon.hflip {
+  transform: scaleX(-1);
+}
+
+.active .icon {
+  filter: brightness(.2);
+}
+
+.icon-read-aloud {
+  width: 20px;
+  height: 16px;
+  background-image: url("/static/icons/read-aloud.svg");
```

Comment:
I was getting 404s for all of the SVGs when I tried to run the demo page.

Reply:
Forgot about the static page. I'd like to keep it around since I still want to replace the page slider with a native one, but I'll have to update it to use the base CSS now.